src/main/java/com/eppo/sdk/EppoClient.java

```diff
@@ -179,12 +176,18 @@ private Optional<EppoValue> determineAndLogBanditAction(VariationAssignmentResul
         if (this.eppoClientConfig.getBanditLogger() != null) {
             // Do bandit-specific logging
 
+            String modelVersionToLog = "cold start"; // Default model "version" if we have not seen this bandit before or don't have model parameters for it
```

Comment:
Open to other default model "version" names where we don't have a model version. For example, we could leave `null` (which I don't like as it's less explicit) or call it something like `uninitialized`.

`cold start` was the best default I could come up with 🤷‍♂️

Reply:
Sounds good to me!